### ISSUE TITLE
Multi row layout custom layout configurator should be a separate section

### ISSUE THREAD

**Issue by mddragnev** [IgniteUI/igniteui-docfx#3623]
Open https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/multi-row-layout#layout-configurator and look at the section navigator. 
![Screenshot 2023-05-03 at 15 19 36](https://user-images.githubusercontent.com/37667452/235913656-e85bcc37-448e-4feb-b837-4da33e25ca83.png)
The layout configurator should be a separate section and not in the Keyboard navigation
